lib/td/client/api/job.rb

```diff
@@ -287,6 +248,114 @@ def query(q, type=:hive, db=nil, result_url=nil, priority=nil, retry_limit=nil,
 
   private
 
+  def validate_content_length(response, current_total_chunk_size)
+    if expected_size = response.header['Content-Range'][0]
+      expected_size = expected_size[/\d+$/].to_i
+    elsif expected_size = response.header['Content-Length'][0]
+      expected_size = expected_size.to_i
+    end
+
+    if expected_size.nil?
+    elsif current_total_chunk_size < expected_size
+      # too small
+      raise EOFError, 'httpclient IncompleteError'
```

Comment:
Is `EOFError` intentional?
Looks `raise IncompleteError` is the correct same as in [api.rb:333](https://github.com/treasure-data/td-client-ruby/pull/85/files#diff-a6ddea27a85b9d434ec8dc459345186fR333) (but I'm not sure..)